exercises/protein-translation/canonical-data.json

```diff
@@ -0,0 +1,143 @@
+{
+  "exercise": "protein-translation",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Translate input RNA sequences into proteins",
+      "comments": [
+        " Returns the name of the protein if given RNA is valid, "
+      , " else throws an error. "
+      ],
+      "cases": [
+        {
+          "description": "Methionine RNA sequence is identified",
+          "property": "translate_codon",
```

Comment:
Lately, we have tried to have move away from testing intermediate results in the canonical data, and only test end results. For this canonical data, I would suggest to do the same and only have a single `translate` property to test. The existing `translate_codon` property tests (like this one) would then use the `translate` property and have as its expected value an array with a single value.

Reply:
I didn't think of this, but it makes much more sense. I'll work on that.